velox/docs/functions/spark/coverage.rst

```diff
@@ -82,7 +82,7 @@
     Scalar Functions                                                                                                                                                                                                           Aggregate Functions                            Window Functions
     =====================================================================================================================================================================================================================  ==  =========================================  ==  =========================================
     :spark:func:`abs`                          count_if                                   inline                                     nvl                                        sqrt                                           any                                            cume_dist                                
-    :spark:func:`acos                          count_min_sketch                           inline_outer                               nvl2                                       stack                                          approx_count_distinct                          dense_rank                               
+    :spark:func:`acos`                         count_min_sketch                           inline_outer                               nvl2                                       stack                                          approx_count_distinct                          dense_rank                               
```

Comment:
Is this a fix for the documentation? Maybe we can do that is a separate PR.

Reply:
Reverted this line in new iteration.